src/expr/node_manager.cpp

```diff
@@ -531,7 +531,7 @@ TypeNode NodeManager::mkBagType(TypeNode elementType)
   CheckArgument(elementType.isFirstClass(),
                 elementType,
                 "cannot store types that are not first-class in bags. Try "
-                "option --uf-ho.");
+                "parsing option --hol.");
```

Comment:
These actually are all outdated.  The NodeManager should never reference parsing options, I'd delete all these checks, which should be done at the API anyways.